tests/exception_handling/asynchronous_exceptions.cpp

```diff
@@ -21,15 +21,27 @@ void check_exception_list_types() {
   static_assert(std::is_same<value_type, std::exception_ptr>::value,
                 "exception_list::value_type is of wrong type");
 
-  { check_type_existence<sycl::exception_list::reference> typeCheck; }
-  { check_type_existence<sycl::exception_list::const_reference> typeCheck; }
+  {
+    check_type_existence<sycl::exception_list::reference> typeCheck;
+    (void)typeCheck;
```

Comment:
Just curious: do we need these `(void)` to silent something here?
Executing the constructor and destructor is not enough?

Reply:
Yes, it's here to silent warnings for unused variable.
Do you think it's better to add comment clarifying it?

Reply:
OK.
Then perhaps this API is too low level and we could have instead just a `constexpr` function
```c++
check_type_existence<sycl::exception_list::reference>();
```

Reply:
Thank you. Done in https://github.com/KhronosGroup/SYCL-CTS/pull/160/commits/3c163d7f918c9fa995a191ee52101e1a2aceb4ff